app/models/journeys/Journey.scala

```diff
@@ -157,12 +157,12 @@ object Journey extends Enum[Journey] with utils.PlayJsonEnum[Journey] {
     override val pageKeys: List[PageName] = List(
       ZeroEmissionCarsPage.pageName,
       ZecAllowancePage.pageName,
-      ZecTotalCostOfCarPage.pageName,
-      ZecOnlyForSelfEmploymentPage.pageName,
-      ZecUseOutsideSEPage.pageName,
-      ZecUseOutsideSEPercentagePage.pageName,
-      ZecHowMuchDoYouWantToClaimPage.pageName,
-      ZecClaimAmount.pageName
+      ZegvTotalCostOfVehiclePage.pageName,
+      ZegvOnlyForSelfEmploymentPage.pageName,
+      ZegvUseOutsideSEPage.pageName,
+      ZegvUseOutsideSEPercentagePage.pageName,
+      ZegvHowMuchDoYouWantToClaimPage.pageName,
+      ZegvClaimAmountPage.pageName
```

Comment:
These should be in GoodsVehicle, not cars. Strangely, all the ones for vehicles have disappeared outside of this PR